src/frontend/komponenter/Fagsak/Vedtak/Vedtaksmeny.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import styled from 'styled-components';
+
+import { ExpandFilled } from '@navikt/ds-icons';
+import { Button } from '@navikt/ds-react';
+import { Dropdown } from '@navikt/ds-react-internal';
+import { NavdsSpacing10 } from '@navikt/ds-tokens/dist/tokens';
+
+import { useBehandling } from '../../../context/behandlingContext/BehandlingContext';
+import type { IBehandling } from '../../../typer/behandling';
+import KorrigerEtterbetaling from './KorrigerEtterbetaling/KorrigerEtterbetaling';
+import EndreEndringstidspunkt from './VedtakBegrunnelserTabell/EndreEndringstidspunkt';
+
+interface IVedtakmenyProps {
+    åpenBehandling: IBehandling;
+    erBehandlingMedVedtaksbrevutsending: boolean;
+}
+
+const KnappHøyreHjørne = styled(Button)`
+    position: absolute;
+    top: ${NavdsSpacing10};
+    right: ${NavdsSpacing10};
+`;
+
+const StyledDropdownMeny = styled(Dropdown.Menu)`
+    width: 36ch;
```

Comment:
Burde denne vera rem? Ser ikkje ut som vi bruker ch elles.

Reply:
Vanligvis enig i at vi alltid bruker rem! Men - dette elementet har `width: 27ch;` i designsystemet, og det ble litt trangt for innholdet i vedtaksmenyen vår 😅 Tenker at `ch` gir mening her, ved at vi bare overstyrer antallet og ikke enheten
https://github.com/navikt/Designsystemet/blob/master/%40navikt/internal/css/dropdown.css#L5